src/presentation/api/helper/serviceResponseWrapper.go

```diff
@@ -34,3 +34,12 @@ func ServiceResponseWrapper(
 	}
 	return c.JSON(responseStatus, formattedResponse)
 }
+
+func ServiceTokenResponseWrapper(
```

Comment:
There is no need to create an auxiliary method called ServiceToken. What that even means? If you just want to set a header, just set on the controller. The context of Echo is passed to ServiceResponse.